### PR TITLE
MF-143:Improve Height and Weight Navigation and Structure

### PR DESCRIPTION
**Pull request by kagai**
WIP

### CODE REVIEW

**Comment by kagai** [comment]
At the moment the record component does not display the data, still waiting for the endpoint



hw
heightweight1

**Review by denniskigen — APPROVED** [review]
Nice work @kagai!

File: src/widgets/heightandweight/heightandweight-record.component.tsx [hunk 1]
```diff
@@ -51,7 +55,7 @@ export default function HeightAndWeightDetailedSummary(
       <div className={styles.heightAndWeightDetailedSummary}>
         <SummaryCard
           name="Height & Weight"
-          editBtnUrl={`/patient/dimensions`}
+          editBtnUrl={``}
```
Comment (donaldkibet) [inline src/widgets/heightandweight/heightandweight-record.component.tsx]:
```suggestion

```

File: src/widgets/heightandweight/heightandweight-summary.component.tsx [hunk 2]
```diff
@@ -85,9 +85,7 @@ function HeightAndWeightSummary(props: HeightAndWeightSummaryProps) {
                   {dimension.bmi || "\u2014"}
                 </td>
                 <td style={{ textAlign: "end" }}>
-                  <Link
-                    to={`?tab=Height%20and%20Weight&uuid=${dimension.obsData.weight.uuid}`}
-                  >
+                  <Link to={``}>
```
Comment (donaldkibet) [inline src/widgets/heightandweight/heightandweight-summary.component.tsx]:
```suggestion

```
why do we have this link here?

**Review by donaldkibet — APPROVED** [review]
Nice work, just a few comments

**Pull request merged**